packages/integration-tests/index.js

```diff
@@ -616,14 +763,19 @@ async function rulesList(stackName, bucketName, rulesDirectory) {
  * @returns {Promise.<number>} - Number of rules deleted
  */
 async function deleteRules(stackName, bucketName, rules, postfix) {
+  // setProcessEnvironment is not needed by this function, but other code
+  // depends on this undocumented side effect
   setProcessEnvironment(stackName, bucketName);
-  const promises = rules.map((rule) => {
-    if (postfix) {
-      rule.name += postfix;
-    }
-    return limit(() => _deleteOneRule(rule.name));
-  });
-  return Promise.all(promises).then((rs) => rs.length);
+
+  process.env.RulesTable = `${stackName}-RulesTable`;
```

Comment:
It seems like this is already set by `setProcessEnvironment`?

Reply:
See https://github.com/nasa/cumulus/pull/1475/files#r382790636